docs/chromadb-schema.md

```
@@ -0,0 +1,172 @@
+    "distance": 0.3524
+  },
+  {
+    "content": "最近仕事でミスが続いて落ち込んでいるんだ...",
+    "distance": 0.4102
+  }
+]
+```
+
+- **content**: ドキュメントの内容 (`documents`)
+- **distance**: クエリとの距離 (値が小さいほど類似度が高い)
+    - 0.0 は完全一致
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)